util/logger.js

```diff
@@ -0,0 +1,21 @@
+const winston = require('winston');
+
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.json(),
+  defaultMeta: { service: 'user-service' },
+  transports: [
+    new winston.transports.Console({ colorize: true }),
+    new winston.transports.File({
+      filename: './logs/error.log',
+      level: 'error',
+      maxsize: '10000000'
+    }),
+    new winston.transports.File({
+      filename: './logs/combined.log',
+      maxsize: '100000000'
```

Comment:
I would write a small comment here explains this number, something like 
```js
maxsize: '100000000' //100 mb
```

Reply:
Got it, will do.